lookout/core/garbage_exclusion.py

```diff
@@ -0,0 +1 @@
+GARBAGE_PATTERN = r"^.*[.-]min\.js"
```

Comment:
We should gather this pattern by "|"-ing the patterns for each supported language.

Reply:
I'll join a list of patterns but separating them explicitely in different languages shouldn't be done IMO to reflect the fact that lookout cannot filter based on language.

Reply:
Sure. This is done to explicitly force new language developers to think about exclusion patterns and define them in each language package